src/colorizer/ColorizeCanvas2D.ts

```diff
@@ -214,21 +229,35 @@ export default class ColorizeCanvas2D implements IRenderCanvas {
     this.pickScene.add(this.pickMesh);
 
     // Configure track lines
-    this.points = new Float32Array([0, 0, 0]);
+    this.lineBufferSize = INITIAL_TRACK_PATH_BUFFER_SIZE;
+    this.linePoints = new Float32Array(this.lineBufferSize);
+    this.lineColors = new Float32Array(this.lineBufferSize);
+    this.lineIds = [];
 
     const lineGeometry = new LineGeometry();
-    lineGeometry.setPositions(this.points);
+    lineGeometry.setPositions(this.linePoints);
```

Comment:
is this necessary to make `lineGeometry` renderable? this is now a not-insignificantly-sized buffer

Reply:
Line geometry needs to be initialized with *some* starting value... I could totally make the initial buffer 0 though and resize it from there?